build-system/compile/compile.js

```diff
@@ -184,13 +184,14 @@ function compile(
       const polyfills = fs.readdirSync('src/polyfills');
       const polyfillsShadowList = polyfills.filter((p) => {
         // custom-elements polyfill must be included.
-        return p !== 'custom-elements.js';
+        return !['custom-elements.js', 'intersection-observer.js'].includes(p);
```

Comment:
I suppose let's update the comment above and add a bit more info there?

Reply:
done